xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-filters/xwiki-platform-notifications-filters-api/src/main/java/org/xwiki/notifications/filters/internal/scope/ScopeNotificationFilterPreference.java

```diff
@@ -126,6 +126,22 @@ public boolean isParentOf(ScopeNotificationFilterPreference other)
                 && other.getScopeReference().hasParent(this.getScopeReference());
     }
 
+    /**
+     * @return true if the current filter preference is a potential parent of other preferences, false otherwise.
```

Comment:
Even if it's internal, it's still interesting to have a `@since`.

Reply:
Done.